src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const transformedState = [];
```

Comment:
```suggestion
  const transformedStates = [];
```
this is plural, all states, so use meaningful names